packages/account/src/Helpers/__tests__/utils.spec.ts

```diff
@@ -0,0 +1,165 @@
+import {
+    generatePlaceholderText,
+    documentAdditionalError,
+    shouldShowIdentityInformation,
+    getDocumentData,
+    getRegex,
+    preventEmptyClipboardPaste,
+    isFieldImmutable,
+} from '../utils';
+
+describe('generatePlaceholderText', () => {
```

Comment:
I think its better to have separate files for different test suites. Because it will make file more lengthy and readabilityof test case is also affected.
cc: @farzin-deriv

Reply:
The test suits over here have one or 2 test cases testing functionality of utility functions. I dont think we need to have individual files for test suits. The only reason for creating test suits here was to group the testcases for each function of the util file